meson.build

```diff
@@ -312,12 +312,6 @@ if build_arping == true
 		dependencies : [rt_dep, cap_dep, idn_dep, intl_dep],
 		link_with : [libcommon],
 		install: true)
-	meson.add_install_script('build-aux/setcap-setuid.sh',
```

Comment:
That's  too harsh. I would rather see NO_SETCAP_OR_SUID default to true.

```
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -47,7 +47,7 @@ option('BUILD_MANS', type : 'boolean', value : true,
 option('BUILD_HTML_MANS', type : 'boolean', value : false,
        description : 'Build html manuals')
 
-option('NO_SETCAP_OR_SUID', type : 'boolean', value : false,
+option('NO_SETCAP_OR_SUID', type : 'boolean', value : true,
        description : 'Allow disabling build-aux/setcap-setuid.sh')
```

Reply:
But you definitely need it for `ping` and probably good idea to have it for `clockdiff` (both shouldn't have security constraints). Therefore `NO_SETCAP_OR_SUID` is good for distros (packaging tools usually do it with their own tools), but not for users compiling from sources. I was thinking to have also switch for each of these 3 binaries, maybe that'd be the way.

Reply:
Good point. Perhaps setcaps / setuid should be configurable per application, and arping & clockdiff should default to no extra bits.

While fixing that it would be great to see better variable name. The NO_... leads to weird disable enabling. Just dropping the NO_ prefix, and flipping meaning of true and false will make that much better.

Reply:
@kerolasa how about now?

Reply:
@kerolasa `traceroute6` permissions does not work. I need to look into it more (even `cap_net_raw+pe` does not work)